python/ray/tests/spark/test_basic.py

```diff
@@ -176,7 +175,7 @@ def f(x):
 
             assert "RAY_ADDRESS" not in os.environ
 
-            time.sleep(7)
+            time.sleep(120)
```

Comment:
why is that, this will make the test too long to finish, we should avoid sleep if possible

Reply:
It have to wait all Ray node exiting, so it takes a bit longer.